.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Potential Bug", "To Triage"]
+assignees: achakko
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please contact the Matomo community forum for questions: https://forum.matomo.org/
+        If this bug is regarding a premium plugin you purchased on plugins.matomo.org, please email shop@matomo.org instead of creating a bug report here.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please tell us what happens instead of the expected behavior.
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
```

Maybe for the value/placeholder give a better example of what someone might write, guiding people more, same for `what-should-actually-happen`, `steps-to-reproduce`

---

Should we have default values in there at all? That will actually hide the placeholder, so I can't see any benefit in having both.

---

Yep, I agree here with Stefan, the value obscures the placeholder and needs to be removed first. So perhaps the example text should be part of the placeholder, e.g.

```
placeholder: Tell us what you see, e.g. "An error messaged shows when I visit Overview under Visitors"
```

---

Thanks for the feedback. Updated placeholder to include a good example.